core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -3961,6 +3961,42 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-aws-3
     namespace: ci
+- from:
+    .awscred:
+      field: .awscred
+      item: serverless-aws-credentials
```

Comment:
I'm not exactly sure what these items correspon to. I have followed instructions from https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/#providing-credentials and created a secret collection named "serverless-aws-credentials" with keys like in the picture below:
![image](https://github.com/openshift/release/assets/336823/e980210e-c3c4-4bc0-b339-a7a93c3abf45)